region/brinstar/green/Green Brinstar Main Shaft.json

```diff
@@ -4708,14 +4708,28 @@
       "entranceCondition": {
         "comeInWithSidePlatform": {
           "platforms": [
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 45,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [],
+              "note": "Applies to Halfie Climb Room."
+            },
             {
               "minHeight": 3,
               "maxHeight": 3,
               "minTiles": 39.4375,
               "speedBooster": true,
               "obstructions": [[3, 2]],
               "requires": [],
-              "note": "Applies to Metal Pirates Room."
+              "note": "Applies to Metal Pirates Room.",
```

Comment:
Springball bouncing is easier for both.  

In metal pirates you have to do a trickydashjump from left wall->~8 tiles from the right wall.

Reply:
I started looking into adding a spring ball bounce strat now and then realized that there's an in-room tricky dash jump into spring ball jump strat that's much easier than either of these :man_facepalming: Added a `devNote` now to mention it. Should we keep the side platform strats with that, or just remove them?

Reply:
Springball Bounce halfie climb is a fine alternative to trickydashjump.  
in room trickydashjump is simpler than the out of room trickydashjump.  I could see disabling trickyDashJump but not canInsaneJump, but the in room jump is very reasonable.

Reply:
Sorry for the delay on this, been distracted with ALTTP editor stuff. Just added a cross-room spring ball bounce strat, because yeah it is a reasonable alternative to the tricky dash jump. I was going to put `canTrickySpringBallBounce` (Expert) on it, but testing it out it seems simple/lenient enough that this doesn't seem needed; so it's just a `canTrickyJump`, putting it into Very Hard if that sounds good?